meta-lmp-base/recipes-support/mfgtool-resize-helper/mfgtool-resize-helper_0.1.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Rootfs disk resize-helper for mfgtools"
+SECTION = "devel"
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-2-Clause;md5=cb641bc04cda31daea161b1bc15da69f"
+
+inherit allarch
+
+RDEPENDS_${PN} += "udev e2fsprogs-resize2fs gptfdisk parted util-linux-findmnt"
```

Comment:
Do we need findmnt? We might also need e2fsck here (the package that provides it).